src/DateSelector/Aside.js

```diff
@@ -0,0 +1,123 @@
+import React, { Component } from 'react'
+import PropType from 'prop-types'
+import {
+  evolve,
+  has,
+  map,
+  merge,
+  pipe,
+  uncurryN,
+  when,
+} from 'ramda'
+
+import ThemeConsumer from '../ThemeConsumer'
+
+const consumeTheme = ThemeConsumer('UIDateSelector')
+
+const addPropIfHasDate = prop => when(
+  has('date'),
+  merge(prop)
+)
+
+const addProp = uncurryN(2, property => map(
+  pipe(
+    addPropIfHasDate(property),
+    evolve({
+      list: map(addPropIfHasDate(property)),
+    })
+  )
+))
+
+class Aside extends Component {
+  renderRadio ({ date, key, label }) {
+    const {
+      name,
+      selectedPreset,
+    } = this.props
+
+    const id = `${name}${key}`
+
+    return (
+      <li key={`${key}${label}`}>
```

Comment:
could you create a variable like make with `const id`?
```suggestion
      <li key={key}>
```